src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, AreaCalculator {
+    private String name;
+    private double area;
```

Comment:
this property is calculated based on other properties, no need to have it separately
```suggestion
```